src/main/java/Triangle.java

```diff
@@ -0,0 +1,56 @@
+public class Triangle extends Figure {
+
+    private int legFirst;
+    private int legSecond;
+    private double hypotenuse;
+
+    public Triangle() {
+        createRandomFigure();
+    }
+
+    public Triangle(String color, int legFirst, int legSecond) {
+        this.legFirst = legFirst;
+        this.legSecond = legSecond;
+        hypotenuse = hypotenuseCalculation(legFirst, legSecond);
+        setArea(areaCalculation(legFirst, legSecond));
+        setColor(color);
+    }
+
+    public int getLegFirst() {
+        return legFirst;
+    }
+
+    public int getLegSecond() {
+        return legSecond;
+    }
+
+    public double getHypotenuse() {
+        return hypotenuse;
+    }
+
+    public void createRandomFigure() {
+        String[] colors = {"green", "blue", "yellow", "black"};
+        int indexColor = (int) (Math.random() * 4);
+        setColor(colors[indexColor]);
+        legFirst = (int) (Math.random() * 50) + 1;
+        legSecond = (int) (Math.random() * 50) + 1;
+        hypotenuse = hypotenuseCalculation(legFirst, legSecond);
+        setArea(areaCalculation(legFirst, legSecond));
+    }
```

Comment:
1. You shouldn't have such method in your class https://www.tutorialspoint.com/design_pattern/factory_pattern.htm
2. this method is void, so how were you going to use it?